pages/k8s/install-aws.md

```diff
@@ -0,0 +1,266 @@
+---
+wrapper_template: "base_docs.html"
+markdown_includes:
+  nav: "shared/_side-navigation.md"
+context:
+  title: "CDK on AWS"
+  description: Running CDK on AWS using the aws-integrator.
+keywords: aws, integrator, ebs, elb
+tags: [install]
+sidebar: k8smain-sidebar
+permalink: using-aws.html
+layout: [base, ubuntu-com]
+toc: False
+---
+
+## AWS integrator
+
+The `aws-integrator` charm simplifies working with **CDK** on AWS. Using the
+credentials provided to Juju, it acts as a proxy between CDK and the underlying cloud,
```

Comment:
Technically, you can give it different creds than the ones Juju is using if you use charm config rather than `juju trust`.  Not sure if that's worth muddying the waters over, though.

Reply:
I added a note and link to the charm store page for more config details :+1: